go/examples/simple_pub/simple_pub.go

```diff
@@ -43,7 +43,7 @@ func main() {
 
 	node := dps.CreateNode("/", keyStore, nodeId)
 	dps.StartNode(node, dps.MCAST_PUB_ENABLE_SEND, nil)
-	fmt.Printf("Publisher is listening on port %v\n", dps.GetPortNumber(node))
+	fmt.Printf("Publisher is listening on %v\n", dps.NodeAddrToString(dps.GetListenAddress(node)))
```

Comment:
This combo appears often enough that is probably deserves an API - something like dps.GetPrintableListenAddress()